pkg/database/migrations/20250603170830_init.sql

```diff
@@ -0,0 +1,62 @@
+
+-- +goose Up
+-- +goose StatementBegin
+-- Create the main table for TestWorkflowExecution
+CREATE TABLE test_workflow_executions (
+    id VARCHAR(255) PRIMARY KEY,
+    group_id VARCHAR(255),
+    runner_id VARCHAR(255),
+    runner_target JSONB,
+    runner_original_target JSONB,
+    name VARCHAR(255) NOT NULL,
+    namespace VARCHAR(255),
+    number INTEGER,
+    scheduled_at TIMESTAMP WITH TIME ZONE,
+    assigned_at TIMESTAMP WITH TIME ZONE,
+    status_at TIMESTAMP WITH TIME ZONE,
+    signature JSONB,
+    result JSONB,
+    output JSONB,
+    reports JSONB,
+    resource_aggregations JSONB,
+    workflow JSONB,
+    resolved_workflow JSONB,
```

Comment:
I know this is all still early draft. But putting some thoughts down while looking at this PR. 

Workflow and resolved_workflow are **HUGE** json documents. It could be beneficial to experiment with extracting those to a separate table.

I also guess we want to re-use all of this code in the control plane like we did in the past? In this case, it might be worth to also introduce some mechanism to avoid needing table per org. For instance, a `tenant_id` which for open source is always the same constant and the org_id for control plane. We'd still need a second "abstract" constant. I guess `sub_tenant_id` does not really work 😋 Maybe the tenant_id would simply be org_id and env_id concatenated? Anyway I'm definitely running a bit ahead of this project here.

Reply:
Sure, it's just a first pass. I will keep it mind on next iterations!

Reply:
And thank you for earlier review!

Reply:
ask AI to split to multiple tables

Reply:
will that make it hard to migrate data from standalone to control-plane?